packages/algob/src/internal/deployer_cfg.ts

```diff
@@ -1,7 +1,7 @@
 import { mkAccountIndex } from "../lib/account";
 import { AlgoOperator } from "../lib/algo-operator";
 import { loadASAFile } from "../lib/asa";
-import { loadCheckpointsRecursive } from "../lib/script-checkpoints";
+import { CheckpointRepoImpl } from "../lib/script-checkpoints";
 import type {
```

Comment:
what is the difference between `loadCheckpointsRecursive ` and `CheckpointRepoImpl` ?

Reply:
`CheckpointRepoImpl` is a raw checkpoint implementation with empty data. `loadCheckpointsRecursive` is returning the cp impl with loaded cp data(`asa`, `lsig`..) of the previous deployed scripts.

Reply:
ok